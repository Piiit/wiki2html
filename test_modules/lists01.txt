~~NOTOC~~
* item 1
* **bold item 2**
* item 3
FINISHED
