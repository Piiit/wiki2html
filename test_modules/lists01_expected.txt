<ul>
<li>item 1</li>
<li><b>bold item 2</b></li>
<li>item 3</li>
</ul>
FINISHED
